### DISCUSSION TITLE
Problem signing up on my new server

### DISCUSSION THREAD

**Discussion by jesserizzo** [mastodon/mastodon#21903] [Q&A]
I'm having problems signing up on my brand new instance. I've only got a single admin user so far. When I try and sign up a new user as a test, the request times out and I get a generic "We're sorry, but something went wrong on our end" error screen. BUT, if I wait a minute or two, the new account is actually created. Thanks.

**Comment by mjankowski** [comment]
Did you ever work this out?

If still an issue - do you have relevant logs?
